demos/JWST/S3_niriss_template.ecf

```diff
@@ -4,25 +4,25 @@ ncpu	  	 1	       # Number of CPUs
 suffix	  	 calints       # Data file suffix
 
 # Order location estimation
-method		 2		# Which method to identify orders by (see docs)
-save		 False		# Saves a table of central value for each order
+method		 2		    # Which method to identify orders by (see docs)
```

Comment:
Don't these methods now have names?

Reply:
they do! I realized I changed it in 1 place, but not this one. It's been updated